mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
```

Comment:
In the `exports.updateOrderStatus` function, the controller function name is `updateOrderStatus`, but in the route, it's being referenced as `update`. These names should match to avoid any errors.